core/crypto/crypto_store/crypto_store_impl.cpp

```diff
@@ -27,26 +27,47 @@ namespace kagome::crypto {
       return res;
     }
 
-    outcome::result<std::pair<KeyTypeId, store::PublicKey>> parseKeyFileName(
-        std::string_view file_name) {
-      if (file_name.size() < 4 + store::PublicKey::size()) {
-        return CryptoStoreError::WRONG_KEYFILE_NAME;
+    outcome::result<std::string> loadFile(
```

Comment:
Do you thing about rename it to **loadFromFile**? We load not file, just some data from that.

Reply:
Actually it is more like loadFileContent, but I see no difference between loadFileContent and loadFile. Nevertheless, I will rename it to loadFileContent if it makes you a little bit happier =)

Reply:
Our common goal is not only being happy personally, but make best quality product. And being happy too, sure. =)